docs/services/navigation/_index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "The Navigation Service"
+linkTitle: "Navigation"
+description: "The Navigation Service uses GPS to autonomously navigate a rover to user defined endpoints."
+type: docs
+weight: 45
+no_list: true
+# icon: "/services/img/icons/frame-system.svg"
+tags: ["navigation", "services", "base", "rover"]
+# SMEs: Raymond
+---
+
+The Navigation service is the stateful definition of Viam's [Motion service](/services/motion/)
```

Comment:
```suggestion
The Navigation service is the stateful definition of Viam's [Motion Service](/services/motion/)
```

Captialized if we're refering to the concept

Reply:
Done